### ISSUE TITLE
Add org/team roles

### ISSUE THREAD

**Issue by nedredmond** [nedredmond/gh-role#5]

**Comment by nedredmond** [comment]
`MEMBER` or `MAINTAINER`

see: https://docs.github.com/en/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member

schema:
```graphql
"""
The possible team member roles; either 'maintainer' or 'member'.
"""
enum TeamMemberRole {
  """
  A team maintainer has permission to add and remove team members.
  """
  MAINTAINER

  """
  A team member has no administrative permissions on the team.
  """
  MEMBER
}
```

**Comment by nedredmond** [comment]
Resolved by #7
